resources/eventing/README.md

```diff
@@ -3,77 +3,35 @@
 This Helm chart contains all components required for eventing in Kyma:
```

Comment:
Update the Makefile and README.md for the new eventing-backend, to reflect the current implementation:
```
components/eventing-controller/Makefile
components/eventing-controller/README.md
```

Reply:
Will be solved by: https://github.com/kyma-project/kyma/issues/11343